docs/source/models/causes/opioid_use_disorder/index.rst

```diff
@@ -0,0 +1,821 @@
+- **Remission upper limit of 0.2**: Consistent with limits in the dataset
+- **Country-level covariates**:
+
+  - Age-standardized prevalence of intravenous drug use (IDU) for prevalence
+  - Log-transformed estimates of defined daily doses for statistical purposes (SDDD) of prescribed opioid analgesics (consumption per day per million population) for prevalence
+  - Intravenous drug use as covariate for excess mortality rate (EMR)
+
+**Data Sources**
+
+The GBD 2023 estimates incorporate data from systematic reviews and multiple sources:
+
+- **Direct methods**: Population surveys that ask respondents if they use or are dependent on opioids
+- **Indirect methods**: Multiplier methods, back-projection, and capture-recapture methods to indirectly estimate total number of opioid users
+- **IHME-indirect data**: Created by matching government records of people in opioid substitution therapy with literature sources on percentage of people with opioid dependence in treatment, using spatiotemporal Gaussian process regression (ST-GPR) to estimate coverage
+- **Treatment records**: Number of individuals with opioid dependence in substitution therapy (primarily from government sources)
+- **Published literature**: OUD prevalence, incidence, and treatment coverage estimates
+- **Mortality data**: Vital registration systems and verbal autopsy studies
+
+**Data Adjustment**: Direct survey estimates are adjusted upward using MR-BRT (meta-regression—Bayesian, regularized, trimmed) crosswalk to account for under-ascertainment of highly stigmatized opioid use, based on cross-walks to treatment-based estimates. Surveys tend to underestimate the most harmful and stigmatized forms of illicit drug use.
+
+**Severity Distribution**
+
+GBD 2023 estimates the proportion of individuals with opioid use disorder across three severity levels based on data from the US National Epidemiological Survey on Alcohol and Related Conditions (NESARC) and the Comorbidity and Trauma Study. The severity distribution is: asymptomatic 16% (13-19%), mild 37% (20-55%), and moderate/severe 47% (29-64%).
+
+.. list-table:: Severity Distribution for Opioid Use Disorder
+   :widths: 15 15 45 15
+   :header-rows: 1
+
+   * - Severity Level
+     - Proportion (95% UI)
+     - Lay Description
+     - Disability Weight (95% CI)
+   * - Asymptomatic
+     - 16% (13-19%)
+     - Individual meets diagnostic criteria but experiences no current symptoms
+     - 0
+   * - Mild
+     - 37% (20-55%)
+     - Uses heroin (or methadone) daily and has difficulty controlling the habit. When not using, the person functions normally.
+     - 0.335 (0.221-0.473)
+   * - Moderate to Severe
+     - 47% (29-64%)
+     - Uses heroin daily and has difficulty controlling the habit. When the effects wear off, the person feels severe nausea, agitation, vomiting, and fever. The person has a lot of difficulty in daily activities.
```

Comment:
Similar comment on heroin being listed but not other drugs

Reply:
I guess this goes back to survey questions written before prescription and synthetic opioids were recognized as such a big problem.  N.b. it is possible that the stigma of the word "heroin" is the reason for the extremely high DW.